OracleDatabase/README.md

```diff
@@ -135,6 +135,9 @@ Another option is to use `docker exec` and run `sqlplus` from within the same co
 
 	docker exec -ti <container name> sqlplus pdbadmin@ORCLPDB1
 
+## Known issues
+* OverlayFS (`overlay`) on CentOS has proven to run into Docker bug #25409. We recommend using `btrfs` or `overlay2` instead. For more details see issue #317.
```

Comment:
Ok, can the ordinary user differentiate between OverlayFS and the graph drivers? Maybe we can just say "`overlay` on CentOS has proven..."